src/components/Inputs/DynamicSelect/DynamicSelect.tsx

```diff
@@ -0,0 +1,109 @@
+import { Select, InputLabel, OutlinedInput, FormControl, SelectChangeEvent, MenuItem } from "@tracktor/design-system";
+// import ControlledToolTip from "@/components/ControlledToolTip/ControlledToolTip";
+import { useState } from "react";
+import ControlledTooltip from "@/components/ControlledToolTip";
+import useDynamicSelect from "@/components/Inputs/DynamicSelect/useDynamicSelect";
+import ChangeEventField from "@/types/ChangeEventField";
+import Headers from "@/types/Headers";
+import TreeNode from "@/types/TreeNode";
+import { JsonFormValue } from "@/utils/formDataToJSON/formDataToJSON";
+
+interface DynamicSelectProps {
+  node: TreeNode;
+  fieldValues?: JsonFormValue[] | unknown;
+  onChange?(dataAttribute: ChangeEventField): void;
+  errorMessage?: string;
+  headers?: Headers;
+  disabledChildrenField?: boolean;
+}
+
+/**
+ * DynamicSelect component
+ * @param treeValue
+ * @param node
+ * @param onChange
+ * @param errorMessage
+ * @param disabledChildrenField
+ * @param headers
+ * @constructor
+ */
+const DynamicSelect = ({ fieldValues, node, onChange, errorMessage, disabledChildrenField, headers }: DynamicSelectProps) => {
+  const [singleOption, setSingleOption] = useState<string>("");
+  const [multipleOptions, setMultipleOptions] = useState<string[]>([""]);
```

Comment:
Ici il faudrait un array vide `[]` à la place de `[""]` car sinon tu active le shrink sur l'option par défaut alors qu'aucune options est sélectionné. Tu n'aura pas le problème de uncontrolled/controlled car ta value existe quand même par défaut, elle est pas `undefined`

```suggestion
  const [multipleOptions, setMultipleOptions] = useState<string[]>([]);
```

Reply:
effectivement ca marche mieux comme ca :) 
<img width="390" alt="Capture d’écran 2024-03-26 à 10 44 38" src="https://github.com/Tracktor/treege-consumer/assets/51543682/aa5caa42-9d92-4e65-b0fb-bbcd72c6c1da">